pkg/cmd/utils.go

```diff
@@ -133,7 +135,7 @@ func GetFeatureFlagsFromImage(containers []v1.Container, imageName string) ([]st
 	}
 	// Extract the flags from the server feature flag args.
 	var featureString string
-	flags := pflag.CommandLine
+	flags := pflag.NewFlagSet("velero-container-command-flags", pflag.ExitOnError)
```

Comment:
Not too familiar with this lib, could you explain what this does?

Reply:
Making this change to ensure the corresponding unit test can be run repeatedly.

`pflag.CommandLine` adds a reference to a global variable/object of the pflag package (like the static variable for a class in Java), while `pflag.NewFlagSet` always create a new FlagSet object.

Both of them should work fine in the real scenario, since it would be only called once either in `backup-driver install` or `data-manager install`. However, the original case, `pflag.CommandLine`, would fail in unit test when there are repeated calls on the function of `GetFeatureFlagsFromImage`.

Specifically, if `flags` points to a package global object, the following statement,
```
flags.StringVar(&featureString, "features", featureString, "list of feature flags for this plugin")
```
would fail in the second call of `GetFeatureFlagsFromImage` due to redefine option with same name.